collect_app/src/main/java/org/odk/collect/android/application/Collect.java

```diff
@@ -171,16 +171,7 @@ public void setExternalDataManager(ExternalDataManager externalDataManager) {
     }
 
     public String getVersionedAppName() {
-        String versionName = "";
-        try {
-            versionName = getPackageManager()
-                    .getPackageInfo(getPackageName(), 0)
-                    .versionName;
-            versionName = " " + versionName.replaceFirst("-", "\n");
```

Comment:
Yes. Just make sure that there is eventually a space put between the app name and the version name. You can run it to see why that's important.